bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Set labels on Strimzi OLM resources to match addon installation

## Description
When installed as an addon, the RHOSAK OLM bundle resource metadata has the following labels:
```
app.kubernetes.io/component: strimzi-bundle
app.kubernetes.io/part-of: managed-kafka
```
The labels are used by the kas-fleetshard operator as the [selection criteria](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/blob/e9dac185828c830a0c6412c1ceaeb6c4d9565e57/operator/src/main/java/org/bf2/operator/managers/StrimziBundleManager.java#L98-L99) for the Strimzi subscription to detect a pending upgrade, etc. This PR adds the same labels to the resources deployed in standalone mode. The difference was noted in testing the changes for MGDSTRM-10052

## Verification Steps

1. Run KFM with standalone cluster configuration
2. Check that the Strimzi operator's `Namespace`, `OperatorGroup`, `Subscription`, and `CatalogSource` resources have the expected labels, listed above.

## Checklist (Definition of Done)
